frontend/src/__tests__/components/button.test.js

```diff
@@ -0,0 +1,60 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Button } from '../../../components';
+import Colors from '../../../components/colors';
```

Comment:
This colors file is oddly capitalized, please be prepared to explain the various standards of capitalization Javascript and React.

Reply:
fixed
google js style guide says all javascript file names lower case (with - and _ as appropriate)
React component file names begin with Capital letter

Reply:
So then let me ask you, what is a component? Is Colors a component? How do we know if things are React Components? Is it just because we put them in a component directory?